### ISSUE TITLE
[FEATURE] Real-time validation in login and Register form

### ISSUE THREAD

**Issue by AlfiyaSiddique** [agamjotsingh18/pollitup#218]
## Description
Currently, for the register and login form, there is no real-time validation where the user gets a validation message as the user enters the input value. 

## Solution
I want to implement the functionality to display the validation messages below respective fields as the user enters the value and the validation message will disappear as soon as the valid input is given.
This will implement from sending invalid data to the backend it also enhances processing speed in the backend.
I will write a reusable code so that it can be implemented in other forms as well like.

### Screenshots

I want to implement functionality similar to below

## Sample Video

https://github.com/agamjotsingh18/pollitup/assets/86224794/b3d8c27f-da3b-4b82-9ea2-e4db3c710923

## Request
If you find this issue worth implementing please assign this issue to me under gssoc label as I am a gssoc contributor

**Comment by AlfiyaSiddique** [comment]
Hello @Arnabdaz, please review this issue.

**Comment by Arnabdaz** [comment]
> Hello @Arnabdaz, please review this issue.

actually another pr is ongoing for this page once that is merged you can work

**Comment by AlfiyaSiddique** [comment]
> > Hello @Arnabdaz, please review this issue.
> 
> actually another pr is ongoing for this page once that is merged you can work

Okay sure, can you please tell me the PR no.?
so I can monitor it.  Once it is merged then I'll start working on this.

**Comment by Arnabdaz** [comment]
#226
